core/templates/services/rte-helper-modal.component.html

```diff
@@ -32,21 +29,11 @@ <h3>Customize This Component</h3>
     <input class="btn btn-secondary e2e-test-cancel-rich-text-editor oppia-cancel-btn"
            value="Cancel"
            (click)="cancel()">
-    <input *ngIf="!currentRteIsMathExpressionEditor && !currentRteIsLinkEditor"
+    <input *ngIf="componentId"
```

Comment:
Wouldn't there always be a componentId? If so then why do we need the ngIf?

Reply:
done, removed the ngif